lib/screens/qpi/add_qpi.dart

```diff
@@ -1,29 +1,353 @@
+import 'package:admu_student_app/constants/app_colors.dart';
 import 'package:flutter/material.dart';
 
 import 'package:admu_student_app/models/course.dart';
+import 'package:admu_student_app/widgets/input_field.dart';
+import 'package:admu_student_app/widgets/buttons.dart';
+import 'package:admu_student_app/widgets/select_color.dart';
+import 'package:google_fonts/google_fonts.dart';
 
 class AddQPIPage extends StatefulWidget {
-  final bool isEditing;
+  // final bool isEditing;
 
-  // template, to add a new course
-  final int yearNum;
-  final int semNum;
+  // // template, to add a new course
+  // final int yearNum;
+  // final int semNum;
 
-  // for editing
-  final Course course;
+  // // for editing
+  // final Course course;
 
-  AddQPIPage({this.isEditing, this.yearNum, this.semNum, this.course});
+  // AddQPIPage({this.isEditing, this.yearNum, this.semNum, this.course});
 
   @override
   _AddQPIPageState createState() => _AddQPIPageState();
 }
 
 class _AddQPIPageState extends State<AddQPIPage> {
+  bool yearSelected = true;
+  bool semesterSelected = false;
+  bool courseSelected = false;
+  String title = 'Year';
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(),
-      body: Text('add/edit qpi'),
+      appBar: AppBar(
+        title: Text('ADD QPI'),
+        elevation: 0,
+      ),
+      body: Center(
+        child: Container(
+          color: AppColors.PRIMARY_MAIN,
+          padding: EdgeInsets.fromLTRB(16.0, 48.0, 16.0, 96.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Row(
+                children: [
+                  Text(
+                    'Add $title QPI',
+                    style: GoogleFonts.dmSans(
```

Comment:
please start using `Theme.of(context).textTheme...` now instead of hardcoding it
also for colors, (import) `AppColors.PRIMARY_...` the file is in `lib/constants/app_colors.dart`

i don't think that we're using plain white `#FFF` and that it's mostly `GRAY_LIGHT` or `GRAY_DARK`
for the gray variants, `index 0 = primary` `index 1 = alt` `index 2 = light` so in this case, it would be `GRAY_LIGHT[2]` or something similar